docs/cli/data.md

```diff
@@ -58,7 +58,7 @@ reference of the gCloud version of SMT, please refer [here](https://cloud.google
 ## REQUIRED FLAGS
 
      --session=SESSION
-        Specifies the file that you restore session state from.
+        Specifies the file that you restore session state from. This file can be generaed using the [schma](schema.md) sub command.
```

Comment:
typo: generated

Reply:
We should also specify what happens if a session file and source profile are passed.

Reply:
`generated`: Fixed in the recent push.
```
We should also specify what happens if a session file and source profile are passed.
```
Added in the docs for Schema sub command, and also added info logs around it.